src/waypoint.jsx

```diff
@@ -333,13 +352,18 @@ export default class Waypoint extends React.Component {
    * @return {Object}
    */
   render() {
+    if (this.props.children) {
+      return <div ref={this.refElement}>{this.props.children}</div>;
```

Comment:
I think that's fine, but I think it would still be good to roll with `React.Children.only` because it will be more painful to make it more restrictive later than it will be to make it less restrictive if we decide later that we should have gone a different route. What do you think?